gradle.properties

```diff
@@ -10,7 +10,7 @@ android.useAndroidX=true
 android.defaults.buildfeatures.buildconfig=true
 
 # Release information
-versionName=6.33.1
```

Comment:
`m`: should this be merged back?

Reply:
Yes, that's fine - as the beta was actually released, craft will automatically overwrite this anyway on the next release.